packages/awsm-codegen/awsm-codegen.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "AWS botocore code generator"
+maintainer: ["Ashish Agarwal <ashish@solvuu.com>"]
+authors: ["Solvuu"]
+license: "MIT"
+homepage: "https://github.com/solvuu/awsm"
+bug-reports: "https://github.com/solvuu/awsm/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "cohttp"
+  "uri-sexp"
+  "core" {< "v0.16.0"}
+  "core_unix"
+  "ocamlformat"
+  "ocamlgraph"
+  "ppxlib" {>= "0.26"}
+  "re" {>= "1.7.2"}
+  "sedlex" {>= "2.3"}
+  "xmlm"
+  "yojson" {>= "1.6.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```
I will disable the tests since they will try to link against awsm and are not really for this package.